test/functional/test_framework/blocktools.py

```diff
@@ -55,6 +55,10 @@
 # Coinbase transaction outputs can only be spent after this number of new blocks (network rule)
 COINBASE_MATURITY = 100
 
+# Soft-fork activation heights
+CLTV_HEIGHT = 1351
+CSV_ACTIVATION_HEIGHT = 432
```

Comment:
It may have been nice to mention the related BIP numbers (65 `OP_CHECKLOCKTIMEVERIFY`, 112 `CHECKSEQUENCEVERIFY`) here.